### PR TITLE
PAH-383 Add filter option to Project and Car select boxes in new drive form

### PR DESCRIPTION
**Pull request by natalix**

### CODE REVIEW

**Comment by arturtamborski** [comment]
I've tested it locally, looks good to me, though there's still a bit of work to do with resolving conflicts...

**Comment by juliaanholcer** [comment]
I haven't tested it locally but code changes appear to be consistent with the rest of the component.

**Comment by pniedzwiedzinski** [comment]
@natalix do you need help with resolving conflicts? I'd be happy to merge it.

File: frontend/src/views/DriveFormView.vue [hunk 1]
```diff
@@ -47,21 +47,18 @@
     </div>
     <div class="form-group">
       <label>{{ $t('drive_form.project') }}</label>
-      <select
+      <v-select
         v-if="projects.data"
-        v-model="project"
+        @input="syncToLocalStorage"
+        v-model="form.project"
```
Comment (pniedzwiedzinski) [inline frontend/src/views/DriveFormView.vue:53]:
```suggestion
        v-model="project"
```

File: frontend/src/views/DriveFormView.vue [hunk 2]
```diff
@@ -72,21 +69,17 @@
 
     <div class="form-group">
       <label>{{ $t('drive_form.cars') }}</label>
-      <select
+      <v-select
         v-if="cars.data"
-        v-model="car"
+        v-model="form.car"
```
Comment (pniedzwiedzinski) [inline frontend/src/views/DriveFormView.vue:74]:
```suggestion
        v-model="car"
```

**Pull request merged**